File: backend-python/app/modules/system/dto/job_request.py
from pydantic import BaseModel,Field
class JobSaveRequest(BaseModel):
 job_name:str=Field(alias="jobName");cron_expression:str=Field(alias="cronExpression");executor_type:str=Field(alias="executorType");script_file:str=Field(alias="scriptFile");script_args:str|None=Field(None,alias="scriptArgs");status:int|None=None;remark:str|None=None

File: backend-rust/src/modules/system/dto/job_request.rs
use serde::Deserialize;

#[derive(Debug, Clone, Deserialize, utoipa::ToSchema)]
#[serde(rename_all = "camelCase")]
pub struct JobSaveRequest {
    pub job_name: String,
    pub cron_expression: String,
    pub executor_type: String,
    pub script_file: String,
    pub script_args: Option<String>,
    pub status: Option<i32>,
    pub remark: Option<String>,
}
